demonstrations/tutorial_zne_catalyst.py

```diff
@@ -0,0 +1,274 @@
+
+ZNE in particular is also offered out of the box in PennyLane as a *differentiable* error mitigation technique,
+for usage in combination with variational workflows. More on this in the tutorial
+:doc:`Differentiating quantum error mitigation transforms <tutorial_diffable-mitigation>`.
+
+On top of the error mitigation routines offered in PennyLane, ZNE is also available for just-in-time
+(JIT) compilation. In this tutorial we see how an error mitigation routine can be
+integrated in a Catalyst workflow.
+
+At the end of the tutorial, we will compare the execution time of ZNE routines in
+pure PennyLane vs. PennyLane and Catalyst with JIT.
+
+What is zero-noise extrapolation (ZNE)
+-----------
+Zero-noise extrapolation (ZNE) is a technique used to mitigate the effect of noise on quantum
+computations. First introduced in [#temme2017zne]_, it helps improve the accuracy of quantum
+results by running circuits at varying noise levels and extrapolating back to a hypothetical
+zero-noise case. While this tutorial won't delve into the theory behind ZNE in detail (for which we
+recommend reading the `Mitiq docs <https://mitiq.readthedocs.io/en/stable/guide/zne-5-theory.html>`_
+and the references therein), let's first review what happens when using the protocol in practice.
+
+Stage 1: Generating noise-scaled circuits
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+ZNE works by generating circuits with **increased** noise. Catalyst implements the unitary folding
+framework introduced in [#DZNEpaper]_ for generating noise-scaled circuits. In particular, 
+the following two methods are available:
+
+1. **Global folding**: If a circuit implements a global unitary :math:`U`, global folding applies
+   :math:`U(U^\dagger U)^n` for some integer :math:`n`,
+   effectively scaling the noise in the entire circuit.
+2. **Local folding**: Individual gates are repeated (or folded) in contrast with the entire
+   circuit.
+
+Stage 2: Running the circuits
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Once noise-scaled circuits are created, they need to be run! These can be executed on either real
+quantum hardware or a noisy quantum simulator. In this tutorial, we'll use the
+`Qrack quantum simulator <https://qrack.readthedocs.io/>`_, which is both compatible with Catalyst,
+and implements a noise model. For more about the integration of Qrack and Catalyst, see
+the demo :doc:`QJIT compilation with Qrack and Catalyst <qrack>`.
+
+Stage 3: Combining the results
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+After executing the noise-scaled circuits, an extrapolation on the results is performed  
+to estimate the zero-noise limit---the result we would expect in a noise-free scenario. 
+Catalyst provides **polynomial** and **exponential** extrapolation methods.
+
+Defining the mirror circuit
+---------------------------
+
+The first step for demoing an error mitigation routine is to define a circuit. 
+Here we build a simple mirror circuit starting off a unitary 2-design. 
```

Comment:
```suggestion
Here we build a simple mirror circuit starting off a `unitary 2-design <https://en.wikipedia.org/wiki/Quantum_t-design>`__. 
```
Let's include a web reference.